pkg/chartverifier/profiles/profile-default.go

```diff
@@ -0,0 +1,30 @@
+package profiles
```

Comment:
I'd rename this file to `default.go`, as dashes `-` are not common in Go.

Reply:
Renamed